pyproject.toml

```diff
@@ -97,6 +97,10 @@ extra-dependencies = [
 [tool.hatch.envs.types.scripts]
 check = "mypy --install-types --non-interactive {args:src/reprostim tests}"
 
+[tool.hatch.version]
+path = "src/reprostim/__about__.py"
```

Comment:
This line won't do anything when using `versioningit`.

Reply:
ha -- removal of this line seems have fix it!!!